packages/rollup-plugin/test/index.test.js

```diff
@@ -34,31 +36,72 @@ describe('Index', () => {
     expect(plugin.writeBundle).to.be.a('function') 
   })
 
-  it('has a writeBundle function that uploads sourcemaps', async () => {
+  describe('writeBundle', () => {
     const outputOptions = { dir: 'dist' }
     const bundle = { 'index.map.js': {} }
     const sourcemapData = [{ sourcemapFilename: 'index.map.js' }]
-    td.when(isNonProdEnvMock()).thenReturn(false)
-    td.when(extractSourcemapDataFromBundleMock({ outputOptions, bundle })).thenReturn(sourcemapData)
-    td.when(cleanOptionsMock(options)).thenReturn(options)
 
-    const plugin = honeybadgerRollupPlugin(options)
-    await plugin.writeBundle(outputOptions, bundle)
+    it('Uploads sourcemaps', async () => {
```

Comment:
Can you please write rewrite to:
```suggestion
    it('should upload sourcemaps', async () => {
```
just to keep consistency with the rest of the test files?